### DISCUSSION TITLE
Adding a copyright text to generated files

### DISCUSSION THREAD

**Discussion by vyefremov** [dotnet/templating#7122] [General]
Hey

I'm new to templating and trying to build a simple item template to speed up the scaffolding of required files. 

It's pretty simple to generate files. But there is a rule to have a file header in my project. A file header is configured in `stylecop.json`: `settings.documentationRules.copyrightText` and is automatically generated once I create a file in an IDE.

I could pass the `copyrightText` via `symbols` inside the `template.json`. But it's not really convenient.

Is there any "computed" way to generate a file header for files generated with `dotnet new`?

**Comment by sharwell** [comment]
If `dotnet new` supports a file header, I would expect it to be read via the .editorconfig property `file_header_template`. For example:
https://github.com/dotnet/templating/blob/b0a9bbec6c9255b44041b71a5d7c0b5c0e01b231/.editorconfig#L202-L203

I'm not sure if there is a template syntax for explicitly including a header.
